universe/index.js

```diff
@@ -0,0 +1,49 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of vikram.hello
+//
+// Copyright 2021 undefined <undefined>
+//
+// See LICENSE for details
+"use strict";
+const Tp = require("thingpedia");
+const querystring = require("querystring");
+const assert = require('assert');
+
+const tmdbAccess = "https://api.themoviedb.org/3/";
+const movieSearch = "search/movie?api_key=654083fa9e049ea234e4ae94d3e65774&language=en-US&query=";
+
+module.exports = class MovieClass extends Tp.BaseDevice {
+    //constructor taken from Yelp index.js
+    constructor(engine, state){
+        super(engine, state);
+        this.uniqueId = "vikram.hello";
+        this._queryResults = new Map();
+        this._deviceState = new Map ();
+    }
+    get_movieinfo(query) {
```

Comment:
You define only one query called `movie` in the manifest, but you have two `get_` functions here. This is the source of your ImplementationError. There should only be one `get_movie`.